aion/web/elm/src/Update.elm

```diff
@@ -237,6 +258,21 @@ update msg model =
                 }
                     ! []
 
+
+        UpdateRoomForm name value ->
+            let
+                oldPanelData =
+                    model.panelData
+
+                roomForm =
+                    oldPanelData.roomForm
```

Comment:
Why would you name it `oldPanelData` while you are not updating it anyway? Also I would rename `roomForm` to distinguish variable from actual record.